packages/plugin-compiler/src/main/kotlin/io/realm/kotlin/compiler/AccessorModifierIrGeneration.kt

```diff
@@ -415,16 +431,26 @@ class AccessorModifierIrGeneration(private val pluginContext: IrPluginContext) {
                 //      return backing_field
                 //  }
 
-                // getList/setList gets/sets raw lists soi bypass any converters in accessors
+                // getCollection/setCollection gets/sets raw collections so it bypasses any converters in accessors
                 modifyAccessor(
                     property = declaration,
-                    getFunction = getList,
+                    getFunction = when (collectionType) {
+                        CollectionType.NONE -> TODO()
```

Comment:
Shouldn't this fail with some kind of `IllegalStateException` instead?

Reply:
It shouldn't, this function is only called when working with collections. I will change this so that it is more obvious and also clarify that dictionaries are not supported yet.